Makefile

```diff
@@ -14,13 +14,48 @@ unit-test:
 clean:
 	rm -rf cellxgene_schema_cli/build cellxgene_schema_cli/dist cellxgene_schema_cli/cellxgene_schema.egg-info
 
+# RELEASE HELPERS
+
 pydist: clean
 	cd cellxgene_schema_cli && python setup.py sdist bdist_wheel
 
-test-release: pydist
+# Set PART=[major, minor, patch] as param to make bump.
+# This will create a new release candidate. (i.e. 0.16.1 -> 0.16.2-rc.0 for a patch bump)
+bump-version:
```

Comment:
why not just inline `bump-version` and `bump-release-candidate` into their respective calling targets? Would these ever be called on their own (and not from `create-release{,-candidate}`?)

Reply:
Inlined!